### DISCUSSION TITLE
How do you direct connect with Yggdrasil?

### DISCUSSION THREAD

**Discussion by CommandZomb** [yggdrasil-network/yggdrasil-go#880] [Q&A]
I'm trying to direct connect a game to a friend with yggdrasil. Is there any comprehensive guide to help me? I'm stumped on what to do.

**Comment by d47081** [comment]
Is your game server supports ipv6? 

If yes and your firewall has port opened, your  friend just should [install yggdrasil](https://yggdrasil-network.github.io/installation.html) and provide it address in the client connection. 
Not sure about the ping, it depends from the [public peer](https://publicpeers.neilalexander.dev/) you are connecting to.
